src/components/button.js

```diff
@@ -0,0 +1,43 @@
+import React, { useRef, useEffect } from 'react'
+import { Sampler } from 'tone'
+
+const Button = (props) => {
+
+  useEffect(() => {
+    document.addEventListener('keydown', handleKeydown)
+    window.focus()
+  }, [])
+
+  const { text, audio } = props;
+  const sound = useRef(null);
+
+  const handleKeydown = (e) => {
+    if(e.keyCode === text.charCodeAt()) {
+      sound.current.play();
```

Comment:
also not sure if .play() is going to work the way you want it with this logic

Reply:
it actually works in my local machine... after reviewing the requirements i messed up and im actually playing mp3s from another website... the reason why some don't work is because the promise of playing that sound fails along the way.